src/app/contextBroker/contextBroker.cpp

```diff
@@ -280,6 +280,9 @@ bool            disableMetrics;
 int             reqTimeout;
 bool            insecureNotif;
 
+#ifdef PARANOID_JSON_INDENT
```

Comment:
paranoid ...
Perhaps look for a better name for the define.

JSON_NO_WS, for example

Reply:
The name is not really important, it will be removed soon if everything goes as expected, see https://github.com/telefonicaid/fiware-orion/issues/2760#issuecomment-337334053.

Reply:
Ok

NTC